specification/cluster/get_component_template/ClusterGetComponentTemplateRequest.ts

```diff
@@ -55,6 +55,10 @@ export interface Request extends RequestBase {
      * @server_default false
      */
     flat_settings?: boolean
+    /**
+     * Filter out results, for example to filter out sensitive information. Supports wildcards or full settings keys
+     */
+    settings_filter?: string | string[]
```

Comment:
I considered `Names` instead but @Anaethelion tells me that it's safer to start from an union and maybe move to `Names` or another similar type alias later.